src/components/PuzzleBoard/puzzleBoard.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+class PuzzleBoard extends React.Component {
+  static defaultProps = {
+    puzzle: [],
+    correctlyGuessedLetters: [],
+    specialChars: []
+  };
+
+  renderPuzzle = () => {
+    const { puzzle, correctlyGuessedLetters, specialChars } = this.props;
+    return puzzle.map((letter, index) => {
```

Comment:
This is purely stylistic, but if you wish, you can put this entire map statement directly in the render return method. Same functionality but just different preferences for inlining vs hiding away the logic for rendering.

Reply:
Is it better to have the logic inside the render method? Or does it differ depending on situation?

Reply:
Not necessarily better. Just different styles. Just pointing out another way to do it, but what you have works great